Compfiles/Usa1974P2.lean

```diff
@@ -0,0 +1,143 @@
+/-
+Copyright (c) 2024 David Renshaw. All rights reserved.
```

Comment:
More recently, I've been writing this copyright header as `(c) 2024 The Compfiles Contributors` rather than `(c) 2024 David Renshaw`
I pushed a commit for this file:
https://github.com/dwrensha/compfiles/commit/64bcbacdb4f6a151d0283183b86cb3d1a13c4a3d